libs/datamodel/connectors/datamodel-connector/src/lib.rs

```diff
@@ -21,6 +22,16 @@ pub trait Connector: Send + Sync {
         self.capabilities().contains(&capability)
     }
 
+    fn referential_actions(&self) -> BitFlags<ReferentialAction>;
+
+    fn supports_referential_action(&self, action: ReferentialAction) -> bool {
+        self.referential_actions().contains(action)
+    }
+
+    fn virtual_referential_actions(&self) -> bool {
```

Comment:
I was confused for a few seconds by that one. I think maybe `emulates_referential_actions()` or a doc comment would help.

Reply:
yes. fixing